src/ucm/event/event.c

```diff
@@ -169,8 +170,8 @@ void *ucm_mmap(void *addr, size_t length, int prot, int flags, int fd, off_t off
 {
     ucm_event_t event;
 
-    ucm_trace("ucm_mmap(addr=%p length=%lu prot=0x%x flags=0x%x fd=%d offset=%ld)",
-              addr, length, prot, flags, fd, offset);
+    ucm_trace("ucm_mmap(addr=%p length=%lu prot=0x%x flags=0x%x fd=%d "
+              "offset=%"PRId64 ")", addr, length, prot, flags, fd, offset);
```

Comment:
off_t is not int64_t
better keep as "%ld" and case offset ot long: `..., (long)offset);`